source/clear-linux/guides/maintenance/autospec.rst

```diff
@@ -1,189 +1,249 @@
 .. _autospec:
 
-Build RPMs with autospec
-########################
+autospec
+########
 
-This guide shows you how to create RPMs with :ref:`autospec <autospec-about>`,
-a tool that assists in automated creation and maintenance of RPM packaging
-on |CL-ATTR|.
+**autospec** is a tool to assist in the automated creation and maintenance of
+RPM packaging in |CL-ATTR|. Where a standard RPM build process using
+:command:`rpmbuild` requires a tarball and :file:`.spec` file to start, autospec
+requires only a tarball and package name to start.
 
-See our :ref:`autospec concept page <autospec-about>` for a detailed explaination
-of how ``autospec`` works on |CL|. For a general understanding of how RPMs work,
-we recommend visiting the `rpm website`_ or the `RPM Packaging Guide`_ .
+.. contents::
+   :local:
+   :depth: 1
 
-Prerequisites
-*************
+Description
+***********
 
-This guide requires that you:
+The autospec tool attempts to infer the requirements of the :file:`.spec` file
+by analyzing the source code and :file:`Makefile` information. It will
+continuously run updated builds based on new information discovered from build
+failures until it has a complete and valid :file:`.spec` file. The autospec tool
+makes use of mock to achieve this. It is possible to influence the exact
```

Comment:
mock is a sandbox that autospec creates to do the build.  I don't think i would say that autospec uses mock to do the things ("infer" "analyze" "continuously run update builds") described in previous sentences.  At least that is how I interpret the statement "The autospec tool makes use of mock to achieve this."